packages/terra-site/package.json

```diff
@@ -113,7 +113,8 @@
     "sass-loader": "^6.0.6",
     "style-loader": "^0.19.0",
     "terra-divider": "^1.2.0",
-    "webpack": "^3.6.0",
-    "webpack-dev-server": "2.7.1"
+    "thread-loader": "^1.1.2",
+    "webpack": "^3.10.0",
+    "webpack-dev-server": "2.9.6"
```

Comment:
we need to keep webpack-dev-server at 2.7.1 to keep [IE 10 compatibility](https://github.com/webpack/webpack-dev-server#caveats). We had to bump it back in this PR https://github.com/cerner/terra-core/pull/901

Reply:
oh crap, i remember htat. Thanks